docs/step-by-step/step-02.md

```diff
@@ -0,0 +1,204 @@
+# Step 2. Get to know Netdata's dashboard
+
+Welcome to Netdata proper! Now that you understand how Netdata works, how it's built, and why we built it, you can start
+working with the dashboard directly.
+
+This step-by-step guide assumes you've already installed Netdata on a system of yours. If you haven't yet, hop back over
+to ["step 0"](step-00.md#before-we-get-started) for information about our one-line installer script. Or, view the
+[installation docs](../../packaging/installer) to learn more. Once you have Netdata installed, you can hop back over
+here and dig in.
+
+## What you'll learn in this step
+
+In this step of the Netdata guide, you'll learn how to:
+
+-   [Visit and explore the dashboard](#visit-and-explore-the-dashboard)
+-   [Explore available charts using menus](#explore-available-charts-using-menus)
+-   [Read the descriptions accompanying charts](#read-the-descriptions-accompanying-charts)
+-   [Interact with charts](#interact-with-charts)
+-   [See raised alarms and the alarm log](#see-raised-alarms-and-the-alarm-log)
+
+Let's get started!
+
+## Visit and explore the dashboard
+
+Netdata's dashboard is where you interact with your system's metrics. Time to open it up and start exploring. Open up
+your browser of choice.
+
+If you installed Netdata on the same system you're using to open your browser, navigate to `http://localhost:19999/`. 
+
+If you installed Netdata on a remote system, navigate to `http://HOST:19999/` after replacing `HOST` with the IP address
+of that system. To connect to a virtual private server (VPS), for example, you might navigate to
+`http://203.0.113.0:19999`. We'll learn more on monitoring remote systems and [multiple systems](step-03.md)
+later on.
+
+> From here on out in this tutorial, we'll refer to the address you use to view your dashboard as `HOST`. Be sure to 
+> replace it with either `localhost` or the IP address as needed.
+
+Hit `Enter`. Welcome to Netdata!
+
+![Animated GIF of navigating to the
+dashboard](https://user-images.githubusercontent.com/1153921/63463901-fcb9c800-c412-11e9-8f67-8fe182e8b0d2.gif)
+
+## Explore available charts using menus
+
+**Menus** are located on the right-hand side of the Netdata dashboard. You can use these to navigate to the
+charts you're interested in.
+
+![Animated GIF of using the menus and
+submenus](https://user-images.githubusercontent.com/1153921/63464031-3ee30980-c413-11e9-886a-44594f60e0a9.gif)
+
+Netdata shows all its charts on a single page, so you can also scroll up and down using the mouse wheel, your
+touchscreen/touchpad, or the scrollbar.
+
+Both menus and the items displayed beneath them, called **submenus**, are populated automatically by Netdata based on
+what it's collecting. If you run Netdata on many different systems using different OS types or versions, the
+menus and submenus may look a little different for each one.
```

Comment:
Might make sense to add a comment here mentioning that if you go to the dashboard right after starting Netdata, it may not show all the menus/submenus yet (some collectors take a while to initialize and send in data for the first time, and charts for them won't show up on the dashboard until the first data for them comes in).

Not exactly critical or anything, but I've seen a fair number of people get caught off-guard by this behavior.

Reply:
And ofc amazing which dimension to show `Not Zero` default feature!

Reply:
Was sarcastic actually. I find it very confusing 😄

Reply:
That's kind of my point though. Both the behavior of hiding zero dimensions by default and the delays in populating charts right after startup are points of confusion for users, and explicitly mentioning those behaviors in the tutorial makes it less likely that users will be confused by them.

Reply:
> Both the behavior of hiding zero dimensions by default and the delays in populating charts

There is one more about it. Some internal collectors ignore zero value metrics (they are not collected)

https://docs.netdata.cloud/collectors/cgroups.plugin/#charts-with-zero-metrics

😄